crates/noirc_evaluator/src/ssa_refactor/ssa_builder/mod.rs

```diff
@@ -296,25 +336,23 @@ impl FunctionBuilder {
         self.terminate_block_with(TerminatorInstruction::Return { return_values });
     }
 
-    /// Returns a ValueId pointing to the given function or imports the function
-    /// into the current function if it was not already, and returns that ID.
+    /// Returns a `ValueId` pointing to the given function.
+    ///
+    /// If the function has already been imported, its `ValueId` will be returned,
+    /// else a new `ValueId` will be returned.
     pub(crate) fn import_function(&mut self, function: FunctionId) -> ValueId {
         self.current_function.dfg.import_function(function)
     }
 
-    /// Returns a ValueId pointing to the given oracle/foreign function or imports the oracle
-    /// into the current function if it was not already, and returns that ID.
+    /// Returns a `ValueId` pointing to the given oracle/foreign function.
+    ///
+    /// If the function has already been imported, its `ValueId` will be returned,
+    /// else a new `ValueId` will be returned.
     pub(crate) fn import_foreign_function(&mut self, function: &str) -> ValueId {
         self.current_function.dfg.import_foreign_function(function)
     }
 
-    /// Retrieve a value reference to the given intrinsic operation.
-    /// Returns None if there is no intrinsic matching the given name.
-    pub(crate) fn import_intrinsic(&mut self, name: &str) -> Option<ValueId> {
-        Intrinsic::lookup(name).map(|intrinsic| self.import_intrinsic_id(intrinsic))
-    }
```

Comment:
Why are we removing this function? It looks like it makes `codegen_ident` more complex.